mayastor-test/e2e/common/io_connect_node.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
```

Comment:
Didn't notice this one before. Without `set -e` errors in script are not detected.

```suggestion

set -e

```